libs/solid/profile/src/lib/components/base/base.component.ts

```diff
@@ -207,22 +210,8 @@ export class BaseComponent implements OnInit, AfterViewInit {
       localStorage.getItem('hide_profile_tour') == 'false' ||
       localStorage.getItem('hide_profile_tour') == null
     ) {
-      setTimeout(() => {
-        // need to be modified - not a good solution
-        let category = 'tree-node-category-';
-        let entry = 'tree-node-entry-';
-        for (let i = 0; i < 4; ++i) {
-          category += i;
-          document.getElementById(category)?.click();
-          category = 'tree-node-category-';
-        }
-        for (let i = 1; i < 5; ++i) {
-          entry += i;
-          document.getElementById(entry)?.click();
-          entry = 'tree-node-entry-';
-        }
-      }, 500);
-
+      const location = this.config.profileLocation;
+      this.navigateTo(location.slice(1, location.length - 1));
```

Comment:
das sieht umständlich aus - warum können wir in `profileLocation` nicht direkt den String hinterlegen, der für `navigateTo` gebraucht wird?

Reply:
=.= ich verstehe auch nicht, warum habe ich so gemacht. Ich habe es gerade angeschaut und das fixiert...